src/tests/Microsoft.PowerFx.Interpreter.Tests/PatchFunctionTests.cs

```diff
@@ -27,26 +27,26 @@ public class PatchFunctionTests : PowerFxTest
     {
         private readonly ParserOptions _opts = new ParserOptions { AllowsSideEffects = true };
 
-        [Theory]
-        [InlineData(typeof(PatchFunction))]
-        public async Task CheckArgsTestAsync(Type type)
-        {
-            var expressionError = new ExpressionError()
-            {
-                Kind = ErrorKind.ReadOnlyValue,
-                Severity = ErrorSeverity.Critical,
-                Message = "Something went wrong"
-            };
+        //[Theory]
+        //[InlineData(typeof(PatchFunction))]
+        //public async Task CheckArgsTestAsync(Type type)
+        //{
+        //    var expressionError = new ExpressionError()
+        //    {
+        //        Kind = ErrorKind.ReadOnlyValue,
+        //        Severity = ErrorSeverity.Critical,
+        //        Message = "Something went wrong"
+        //    };
```

Comment:
Why is all this code commented out?

Reply:
I intended to remove this whole file since `PatchFunction` class no longer exists. On second thought, I modified this test case to the new classes. Thanks for pointing this out.